packages/cli/README.md

```diff
@@ -4,10 +4,67 @@ Tool for interacting with the Celo Protocol.
 
 ## Development
 
+### Build
+
 Use `yarn build:sdk <NETWORK>` to build the sdk for the target environment (CLI dependency).
 
 Use `yarn build` to compile the CLI.
 
+### Generate docs
+
 Use `yarn docs` to populate `packages/docs` with generated documentation. Generated files should be checked in, and CI will fail if CLI modifications cause changes in the docs which were not checked in.
 
 _See [@celo/dev-cli](https://github.com/celo-org/dev-cli) for how we customize doc generation._
+
+### Known build issues on Linux
+
+> I'm getting the follow error: `Cannot find module '@celo/contractkit'`.
+
+A possible solution is to build the monorepo manually.
+Go to the `celo-monorepo` root directory and
+
+```bash
+> yarn build
+```
+
+If all works well, navigate to `packages/cli`.
+
+> I'm getting the follow error during the above build: `lerna ERR! yarn run build exited 1 in '@celo/protocol'`.
+
+This error implies that the `truffle` project is not initialized.
+Navigate to `celo-monorepo/packages/protocol` and initialize the truffle project.
+Make sure you do NOT overwrite any existing contracts, migrations or tests.
+
+```bash
+> npm i -g truffle
```

Comment:
this sounds weird...
why would you need to initialize truffle?

Do you have a more specific error log?

Reply:
`node v8.16.1`
`Linux dimi-xps 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux`
`

```
> git clone git@github.com:celo-org/celo-monorepo.git
> yarn
> cd packages/protocol/
> yarn build
```

```
yarn run v1.17.3
$ ts-node ./scripts/build.ts && yarn build:ts
Compiling
Warning: Both truffle-config.js and truffle.js were found. Using truffle-config.js.

Compiling your contracts...
===========================
> Everything is up to date, there is nothing to compile.

{ Error: ENOENT: no such file or directory, open '/home/dimi/celo/celo-monorepo-test/packages/protocol/build/contracts/Proxy.json'
```

Observations:
- Truffle doesn't recognize the `contracts` directory
- Truffle is finding some `truffle-config.js` (there's a few in the `node_modules`, maybe somewhere else on my system)

Reply:
renaming `protocol/truffle.js` to `protocol/truffle-config.js` is a potential fix

Reply:
that sounds more reasonable.
Still is strange that none other has the same issue... are you using `yarn` or `npm`?

Reply:
maybe you can test by adding an empty `truffle-config.js` and try building?

Reply:
sorry a bit confused here

we started by saying that we need to do `truffle init` to make the repo build on linux. That's what i found strange. Surely the truffle-config.js is a result of running `init`, right?

Reply:
I understand the confusion. Forget about the `truffle init` solution.

When I do a vanilla `yarn && yarn build` on the repo I get the error. So without `init`.
Seems that `truffle` finds somewhere `truffle-config.js` it will prefer that one over the existing `truffle.js`. That seems to be the case for me.

Reply:
Not sure if it's an actual issue or just something on my local device due to previous projects using ganache